### ISSUE TITLE
the skin color on changing does not persist

### ISSUE THREAD

**Issue by pratyush1687** [himanish-star/Bumba-The-Assistant-App#15]
initially it's red:
![image](https://user-images.githubusercontent.com/25077017/35158330-c4424f70-fd5c-11e7-8c9f-0e6ecebe58e4.png)

then i change it to indigo:
![image](https://user-images.githubusercontent.com/25077017/35158350-d12bb712-fd5c-11e7-9400-93aa17bea9bc.png)

it reverts to red on opening any other tab from the nav;

![image](https://user-images.githubusercontent.com/25077017/35158391-ec03ab62-fd5c-11e7-9270-79c0bc5bab89.png)

**Comment by shikhar-scs** [comment]
Yeah that's true @pratyush1687 . Thanx for pointing it out though. I'll look into it.

**Comment by pratyush1687** [comment]
I'd like to take it up if that's not a problem

**Comment by shikhar-scs** [comment]
Ofcourse! No problem.
